internal/install/recipe_installer.go

```diff
@@ -294,32 +294,10 @@ func (i *RecipeInstaller) install(ctx context.Context) error {
 		return err
 	}
 
-	if err = i.fetchAndInstallPacks(ctx, recipesForInstall); err != nil {
-		return err
-	}
-
 	log.Debugf("Done installing.")
 	return nil
 }
 
-func (i *RecipeInstaller) fetchAndInstallPacks(ctx context.Context, recipesForInstall []types.OpenInstallationRecipe) error {
-	packs, err := i.packsFetcher.FetchPacks(ctx, recipesForInstall)
```

Comment:
I don't see this used anyplace else, should the packsFetcher code also be removed?